app/views/groups/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1>Categories</h1>
+<% if @groups.any? %>
+    <% @groups.each do |group| %>
+        <div class="row">
+          <div class="col">
+            <%= image_tag group.image %>
+          </div>
+          <div class="col">
+            <h5><%= link_to group.name, group_path(group) %></h5>
+            <p><small><%= group.created_at.strftime("%d-%b-%Y") %></small></p>
+          </div>
+          <div class="col">
+            <p> $<%= group.expenses.sum(:amount) %></p>
+            <%if can? :destroy, group %>
+              <%= link_to 'Remove', group_path(group), class: "remove-btn", method: :delete, data: { turbo_method: :delete, turbo_confirm: "Are you sure you want to delete this group?" } %>
+            <% end %>
+          </div>
+        </div>
+      <% end %>
+      <%= link_to 'Add a new category', new_group_path, class: "btn btn-category" %>
+<% else %>
+  <p class="no-category">There is no category added!!!</p>
+  <%= link_to 'Add a new category', new_group_path, class: "btn btn-category" %>
+<% end %>
```

Comment:
- Kindly ensure that your design matches the original template as much as possible, which includes:
   - [ ] Changing the position of the "Add Category" button to be at the bottom of the screen at a similar position to the green button in the given template.
   - [ ] Editing the background color of the "Add Category" button to match the original design.
   - [ ] Changing the position of the page title to be in the center of the navbar to match the given template.
   (These changes also apply to the transaction page)

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/Nasirkhan294/penny_planner/assets/62290545/f26f55d9-f052-4212-8a5a-59ed42671dac) | ![image](https://github.com/Nasirkhan294/penny_planner/assets/62290545/5b6ccbff-75cb-492e-b66c-873c9eea1c8d) |